google/cloud/vision/entity.py

```diff
@@ -55,7 +55,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = []
+        if 'boundingPoly' in response:
+            bounds = Bounds.from_api_repr(response['boundingPoly'])
```

Comment:
None of the `LABEL_DETECTION_RESPONSE` fixtures supply the `boundingPoly` key, which makes me think it isn't relevant here.  I'm surprised coverage passes, actually, since line 60 won't get exercised.